package.json

```diff
@@ -1,22 +1,29 @@
 {
-    "name": "replaced-by-user-input",
-    "description": "replaced-by-user-input",
+    "name": "minerva",
+    "description": "",
     "version": "0.0.1",
     "private": true,
     "dependencies": {
         "@slack/web-api": "^5.15.0",
+        "aws-cdk-lib": "^2.68.0",
         "aws-sdk": "^2.884.0",
+        "axios": ">=0.21.2",
         "chai": "^4.3.4",
         "chai-as-promised": "^7.1.1",
+        "esbuild": "0.17.10",
         "googleapis": "^66.0.0",
         "lodash": "^4.17.21",
         "mockdate": "^3.0.5",
         "moment": "^2.29.2",
         "moment-timezone": "^0.5.33",
-        "axios": ">=0.21.2",
         "node-notifier": ">=8.0.1"
     },
     "devDependencies": {
+        "eslint": "^8.35.0",
```

Comment:
Added eslint. Will properly set up the linting functionality in a future PR as doing it now will probably cause the linted code to obfuscate the actual change